Sasquatch/SasquatchObjC/Info.plist

```diff
@@ -41,5 +41,18 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>None</string>
+			<key>CFBundleURLName</key>
+			<string>com.microsoft.azure.mobile.sasquatchobjc</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>mobilecenter-3ccfe7f5-ec01-4de5-883c-f563bbbe147a</string>
```

Comment:
Good point, thanks @clpolet. Unfortunately this is right. @bmourat, you need to change it in AppDelegate. Let me see if I can do this.